ros2_kdl_node:
  ros__parameters:
    traj_duration: 8.0
    acc_duration: 5.0
    total_time: 8.0
    trajectory_len: 200
    Kp: 1
    end_position: [0.5, 0.5, 0.5]

